docs/api/utils.rst

```diff
@@ -0,0 +1,604 @@
+
+      :arg  s: any format name from above in upper or lower case, like *"A4"* or *"letter-l"*.
+
+      :rtype: Tuple
+      :returns: *(width, height)* of the paper format. For an unknown format *(-1, -1)* is returned. Examples: *PaperSize("A4")* returns *(595, 842)* and *PaperSize("letter-l")* delivers *(792, 612)*.
+
+-----
+
+   .. method:: PaperRect
+
+      Convenience function to return a :ref:`Rect` for a known paper format.
+
+      :arg string size: any format name supported by :meth:`PaperSize`.
+
+      :rtype: :ref:`Rect`
+      :returns: *Rect(0, 0, width, height)* with *width, height = Utils.PaperSize(size)*.
+
+-----
+
+   .. method:: sRGB2Pdf(int srgb)
+
+      Convenience function returning a PDF color triple (red, green, blue) for a given sRGB color integer as it occurs in :meth:`Page.GetText` dictionaries "dict" and "rawdict".
+
+      :arg int srgb: an integer of format RRGGBB, where each color component is an integer in range(255).
+
+      :returns: a tuple (red, green, blue) with float items in interval *0 <= item <= 1* representing the same color. Example `sRGB2Pdf(0xff0000) = (1, 0, 0)` (red).
+
+-----
+
+   .. method:: sRGB2Rgb(int srgb)
+
+      Convenience function returning a color (red, green, blue) for a given *sRGB* color integer.
+
+      :arg int srgb: an integer of format RRGGBB, where each color component is an integer in range(255).
+
+      :returns: a tuple (red, green, blue) with integer items in `range(256)` representing the same color. Example `sRGB2Pdf(0xff0000) = (255, 0, 0)` (red).
+
+-----
+
+   .. method:: GlyphName2Unicode(string name)
+
+      Return the unicode number of a glyph name based on the **Adobe Glyph List**.
+
+      :arg string name: the name of some glyph. The function is based on the `Adobe Glyph List <https://github.com/adobe-type-tools/agl-aglfn/blob/master/glyphlist.txt>`_.
+
+      :rtype: int
+      :returns: the unicode. Invalid *name* entries return `0xfffd (65533)`.
+
+-----
+
+   .. method:: Unicode2GlyphName(int ch)
+
+      Return the glyph name of a unicode number, based on the **Adobe Glyph List**.
+
+      :arg int ch: the unicode given by e.g. `ord("ß")`. The function is based on the `Adobe Glyph List <https://github.com/adobe-type-tools/agl-aglfn/blob/master/glyphlist.txt>`_.
+
+      :rtype: string
+      :returns: the glyph name. E.g. `Utils.Unicode2GlyphName(Convert.Int32("Ä"))` returns `'Adieresis'`.
+
+-----
+
+   .. method:: PlanishLine(Point p1, Point p2)
+
+      Return a matrix which maps the line from p1 to p2 to the x-axis such that p1 will become (0,0) and p2 a point with the same distance to (0,0).
+
+      :arg Point p1: starting point of the line.
+      :arg Point p2: end point of the line.
+
+      :rtype: :ref:`Matrix`
+      :returns: a matrix which combines a rotation and a translation::
```

Comment:
this is very broken the you view the docs, I think you just need to remove the double "::" as Sphinx then interprets the next block as code.
<img width="765" alt="Screenshot 2024-09-13 at 22 34 05" src="https://github.com/user-attachments/assets/b47cbbac-770f-4a2a-aae4-eea07674b25d">